src/styles/components/_code-block.scss

```diff
@@ -35,8 +35,8 @@ $breakpoint-increase-padding: $breakpoint-scale-up;
       line-height: 2.5rem; // 40px/16
 
       &::after {
-        width: 2.5rem;
-        height: 3rem;
+        width: 2.2rem;
+        height: 2.9rem;
       }
```

Comment:
This adjustments were to make the arrow above the code blocks match [the design](https://www.figma.com/file/UjMQEwUvAsnFAMhYNdUiVT/Apprenticeship-Capstone?node-id=84%3A479&t=PulikziFV3WEVLTu-1) a bit better. The image dimensions are slightly different because the svg in figma has some whitespace, while the new SVG does not.